internal/worker/dbaccessor/tracker_test.go

```diff
@@ -121,7 +123,7 @@ func (s *trackedDBWorkerSuite) TestWorkerAttemptsToVerifyDB(c *gc.C) {
 	defer s.setupMocks(c).Finish()
 
 	s.expectClock()
-	done := s.expectTimer(1)
+	s.expectTimer(1)
```

Comment:
I don't think anything uses the return for this now, so that infra can also be cleaned up.